ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -1292,3 +1292,40 @@ def test_torch_diag(
         input=values[0],
         diagonal=diagonal,
     )
+    
+ 
+@handle_frontend_test(
+    fn_tree="torch.bucketize",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shared_dtype=True,
+        min_num_dims=1,
+        max_num_dims=1,

```

Comment:
Why are we restricting the number of dims to 1 only?

The following is an example from PyTorch's documentation

```
>>> boundaries = torch.tensor([1, 3, 5, 7, 9])
>>> v = torch.tensor([[3, 6, 9], [3, 6, 9]])
>>> torch.bucketize(v, boundaries)
```